core/src/main/java/com/graphhopper/routing/util/parsers/helpers/OSMValueExtractor.java

```diff
@@ -210,7 +215,7 @@ public static double stringToKmh(String str) {
             return Double.NaN;
         }
 
-        if (value <= 0) {
+        if (value < 0) {
```

Comment:
Isn't `maxspeed=0` a tagging error?

Reply:
See #3074. I think we can just ignore anything well below walking speed.

Reply:
A quick google search gave a range of 4 to 6km/h for walking speed. Any value below a "slow walking speed" of, say, 4km/h should be ignored.

Reply:
> Is there any real-world example where speeds are limited for pedestrians?

I'm only aware of limits for motorized traffic.

> Even low maxspeeds that are around walking speeds should be ignored in most cases? 

Let's say there's a road that is limited to walking speed for motorized traffic. The problem is that it's up to the creativity of the tagger how this ends up in OSM. Some people might not know about `maxspeed=walk` and simply tag what's their interpretation of a walking speed. If we ignore values like 4 or 5 km/h just because they don't fit our interpretation of around 6 km/h, we could quickly overestimate the maximum speed on that route.

In essence:

* walk'ish speed of around 4-6 -> normalize to our definition of 6km/h
* anything below -> ignore

The interpretation also varies by country. e.g in the Netherlands it's quite high with 15km/h